server/lib/template/partials/head.handlebars

```diff
@@ -5,6 +5,11 @@
   <link href="{{absolutePath}}/css/bhima-bootstrap.css" rel="stylesheet">
   <script src="{{nodeModulesPath}}/jsbarcode/dist/JsBarcode.all.min.js"></script>
   <style>
+    
+    * {
```

Comment:
After this change, can you removed the `!important` from the handlebars file and it will work?

Reply:
Or will we be required to go into each style tag and put `!important`..?

Reply:
As we dont't know where we set each background <b>transparent !important</b>, adding <b> !important will help

Reply:
How long did you spend looking in the CSS to try and find it?  I would much prefer the solution where we just override that rule, remove it, or change it than asking each developer to always put !important.  That seems like a really silly fix.

Reply:
I'm trying to find that. If it's due to <b>background transparent !important </b> some where in our css I will remote it else we will momentarily add <b>!important</b>

Reply:
Perfect, thank you.  Let me know if you need help finding it.

Reply:
I would like to know where this bellow code is written

/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */
@media print {
  *,
  *:before,
  *:after {
    color: #000 !important;
    text-shadow: none !important;
    background: transparent !important;
    box-shadow: none !important;
  }
```
I can only see it at http://localhost:8080/css/bhima-bootstrap.css

Reply:
It is in bootstrap.less, which imports it from print.less in the node_modules folder.

The exact path is `node_modules/bootstrap/less/print.less`.

Reply:
To override it, you can probably just do:
```less
@media print {
  *,
  *:before,
  *:after {
    color: inherit !important; // Black prints faster: h5bp.com/s
    background: initial !important;
  }
```
Or something similar.  Put that in the bhima-bootstrap.less and it should all work.

Reply:
Curiously works after all those manipulations.

Reply:
Alright, I'll give it a whirl and see what I can come up with.

Reply:
So, the property I was hoping to use ([revert](https://developer.mozilla.org/en-US/docs/Web/CSS/revert)) is not supported in Chrome.  That is the main issue.

Further, since it is applied to all elements (`*`) we cannot make a more specific selector.

So, I think the solution will be to remove the @import bootstrap and create our own @import rules that essentially pull in everything but that `print.less`.  Then we can copy what we want from that print.less file into our own, modified version.

Reply:
@jeremielodi Do you want to tackle this, or do you want me to try?

Reply:
@jniles you can try, but what I noticed is even though you remove all bootstrap less files, the problem remains

```js

/**
 * @function compileLess
 *
 * @description
 * Copiles the less paths into CSS file.  Must be called
 * before the CSS compilation step.
 */
function compileLess() {
  return src(LESS_PATH)
    // .pipe(gulpLess(LESS_CONFIG))
    .pipe(gulpif(isProduction, postcss([cssnano({ zindex : false })])))
    .pipe(dest(`${CLIENT_FOLDER}/css`));
}